service/crt/k8s_secret.go

```diff
@@ -51,3 +51,7 @@ func (s *Service) DeleteCertificate(cert *certificatetpr.CustomObject) error {
 
 	return nil
 }
+
+func (s *Service) getSecretName(cert *certificatetpr.CustomObject) string {
```

Comment:
I would have this as a simple function, since it doesn't need to be a method. Not sure what the most idiomatic Go way is though.